libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -811,8 +811,21 @@
   },
   "admin": {
     "userManagement": {
+      "drawer": {
+        "edit": "Rediger",
+        "active": "aktiv"
+      },
       "topBarTitle": "Brugeradministration",
       "tabs": {
+        "basicdata": {
```

Comment:
suggestion: in the metering point domain, we've called this "master data". I've also heard the term "master data" used when refering to "stamdata". Consider renaming it for consistency but also for using the established terminology within the domain.

*Note: the renaming referts to all places where "basicdata" is used. I only post it once here in order to avoid adding the same comment in different places.